start-deploy.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/bash
+
+set -e
+
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+BASE64="base64 -w 0"
+if [ "${OS}" == "darwin" ]; then
+  BASE64="base64"
+fi
+
+# Helper function to format logs
+function printlog() {
```

Comment:
I'd say this should go in a separate utils file, but there's only one function, so I think it makes sense to just duplicate it. 🙂